src/components/checkboxes/_checkboxes.scss

```diff
@@ -100,4 +100,20 @@
   .govuk-c-checkboxes__input:disabled + .govuk-c-checkboxes__label {
     opacity: .5;
   }
+
+  .govuk-c-checkboxes__conditional {
```

Comment:
is it needed in checkboxes? can/should you have conditional content inside checkbox selection?

Reply:
I'm not 100% sure, but I believe this might be used for contact preferences if the user wants to provide more than one method. Or an opt in checkbox with email input.

Not sure we should actively promote it, but not block it either?

Reply:
as @dashouse says, its good for contact preferences, so I can imagine its good in other situations.